assignment-1/README.md

```diff
@@ -0,0 +1,79 @@
+# assignment-0
+
+# Steps to execute application
+
+- start minikube
+```bash
+$ minikube start
+```
+
+- enable volumesnapshots and csi-hostpath-driver addons
+```bash
+minikube addons enable volumesnapshots
+minikube addons enable csi-hostpath-driver
```

Comment:
@shahpratikr let's assume that this your open source project that you want people to use. Can you please update the readme considering that the above point.